browser/model/jbds-autoinstall.js

```diff
@@ -12,7 +12,7 @@ class JbdsAutoInstallGenerator {
   }
 
   generate(jbdsInstallDir, jdkInstallDir) {
-    let jbdsVersion = "9.1";
+    let pjson = require('../../package.json');

```

Comment:
DevStudio version should be loaded from requirements.json, not from package.json. It also wont work for tests and install time the same way, because root folder is different.